lasagne/layers/pool.py

```diff
@@ -161,14 +162,18 @@ class MaxPool2DLayer(Layer):
         The strides between sucessive pooling regions in each dimension.
         If ``None`` then ``stride = pool_size``.
 
+    ignore_border : bool
+        If ``True``, partial pooling regions will be ignored.
+        Must be ``True`` if ``pad != (0, 0)``.
+
     pad : integer or iterable
         Number of elements to be added on each side of the input
         in each dimension. Each value must be less than
         the corresponding stride.
 
-    ignore_border : bool
-        If ``True``, partial pooling regions will be ignored.
-        Must be ``True`` if ``pad != (0, 0)``.
+    mode : string
+        Pooling mode, one of 'max', 'average_inc_pad' or 'average_exc_pad'.
+        Defaults to 'max'.
```

Comment:
From the [numpydoc HOWTO](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections), this should be:
```
    mode : {'max', 'average_inc_pad', 'average_exc_pad'}
        Pooling mode: max-pooling or mean-pooling including/excluding zeros in
        partial pooling regions. Defaults to 'max'.
```